keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -469,45 +453,42 @@ type LeaseSpec struct {
 For the identity leases, a new lease will be created
 
 ```go
-type IdentityLeaseSpec struct {
+type LeaseCandidateSpec struct {
+  // The fields BinaryVersion and CompatibilityVersion will be mandatory labels instead of fields in the spec
 	// BinaryVersion indicates the binary version of the component
-	BinaryVersion string
-
+	// BinaryVersion string
 	// Compatibility Version indicates the compatibility version of the component. Must be less than or equal to the binary version
-	CompatibilityVersion string
+	// CompatibilityVersion string
+
+	// CanLeadLease is in the format <namespace>/<name>, indicating the namespace and name of the lease that the candidate may lead
+	CanLeadLease string
+
+	// Strategy indicates the preferred strategy for the coordinated leader election controller to use.
+	Strategy CoordinatedLeaseStrategy `json:"strategy,omitempty" protobuf:"string,6,opt,name=strategy"`
```

Comment:
Hmm, this confuses me now. From our Slack thread I didn't expect this field in `LeaseCandidate`. But let me keep reading.